package.json

```diff
@@ -23,34 +23,29 @@
     "babel-cli": "~6.16.0",
     "babel-core": "~6.17.0",
     "babel-eslint": "~7.0.0",
-    "babel-loader": "~6.2.5",
```

Comment:
Please revert this and other changes unrelated to the export feature in this file.